src/gui/Application.cpp

```diff
@@ -142,6 +145,31 @@ Application::~Application()
     }
 }
 
+/**
+ * Perform early application bootstrapping such as setting up search paths,
+ * configuration OS security properties, and loading translators.
+ * A QApplication object has to be instantiated before calling this function.
+ */
+void Application::bootstrap()
+{
+    Bootstrap::bootstrap();
+    Translator::installTranslators();
```

Comment:
How do we translate the CLI? Do we even? This probably should be in Bootstrap::bootstrap()

Reply:
I think we should, yeah. We do use `tr()` in the cli code. The only thing we might want to verify is whether or not we need a `QApplication` to use the translating feature.

Reply:
Yep, the translation is in `QT += core` and we're using a `QCoreApplication` so I'll move it to `Bootstrap::bootstrap()`